### ISSUE TITLE
stop creating directories in the input fileGrp

### ISSUE THREAD

**Issue by bertsky** [mjenckel/ocrd_anybaseocr#60]
Probably a left-over of some failed attempt at temporary file I/O…

https://github.com/OCR-D/ocrd_anybaseocr/blob/287adced2d7963ac074762ec02b435b4a6510cae/ocrd_anybaseocr/cli/ocrd_anybaseocr_dewarp.py#L39

https://github.com/OCR-D/ocrd_anybaseocr/blob/287adced2d7963ac074762ec02b435b4a6510cae/ocrd_anybaseocr/cli/ocrd_anybaseocr_dewarp.py#L158

…these should be removed.

**Comment by bertsky** [comment]
Can also cause failures:
```
FileExistsError: [Errno 17] File exists: 'OCR-D-IMG-DEN-BINPAGE-sauvola-DENOISE-ocropy/test_A/'
```

**Comment by kba** [comment]
```
 os.mkdir(self.input_file_grp+"/test_A/") 
```

This is necessary because pix2pixHD expects this directory, named exactly like that, in that place. The data flow in pix2pixHD's plumbing is so convoluted that it's hard to say without trial&error how to fix this. I have no CUDA hardware, so I cannot test the dewarping unfortunately.

**Comment by bertsky** [comment]
> This is necessary because pix2pixHD expects this directory, named exactly like that, in that place. The data flow in pix2pixHD's plumbing is so convoluted that it's hard to say without trial&error how to fix this.

Understood. Than we should document that, and fix the problem of repeated mkdir (which is now more realistic due to `--overwrite`).

>  I have no CUDA hardware, so I cannot test the dewarping unfortunately.

Too bad!

**Comment by kba** [comment]
fixed by #89
